### PR TITLE
Feature add my growth

### LINKED ISSUE #191: Add calendar / track user progress page
- [ ] Add graph / chart where you can choose a day and it completed tasks
- [ ] Think about how to show the progress
- [ ] Weekly range

### PR DESCRIPTION
**Pull request by conalli**
## Description

Add data visualisation for MyGrowth page

## Closes issue(s)

Closes #191 

## Screenshots

mygrowth-1

## Changes include

- [x] New feature (non-breaking change that adds functionality)

## Checklist

- [x] I have tested this code

### CODE REVIEW

File: src/pages/MyGrowth/components/TabData.tsx [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+      getGardenProgressData(gardenId, startDateISO, endDateISO);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [gardenId]);
+
+  return (
+    <>
+    <LoadingWrapper isLoading={gardenProgressDataApi.status === "loading"}>
+      <Grid container>
+        <Typography className={styles.gardenDescription} variant="h5">This weeks Seeds</Typography>
```
Comment (curtisahuang) [inline src/pages/MyGrowth/components/TabData.tsx:219]: Possessive, so "This Week's Seeds"?

**Review by curtisahuang — APPROVED** [review]
Looks good, 

I'm thinking whether each of the Growth charts needs to have a background to separate them a bit more cleanly. Even something like a very light gray might be nice.

File: src/pages/MyGrowth/components/TabData.tsx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+  const endDate = new Date();
+  
+  const [gardenProgressDataApi, getGardenProgressData] = useApi(
+    getGardenByGardenIdWithDates
+  );
+  // Unused??
+  //   const gardenProgress = useMemo(
+  //   () => gardenProgressDataApi.response?.garden,
+  //   [gardenProgressDataApi]
+  // );
```
Comment (daiHash) [inline src/pages/MyGrowth/components/TabData.tsx:49]: Please remove it if not necessary

**Review by daiHash — APPROVED** [review]
LGTM! Lets merge it and test it locally and check if there somewhere we improve more

**Pull request merged**